frontend/packages/pipelines-plugin/src/components/repository/sections/WebhookSection.tsx

```diff
@@ -38,14 +38,20 @@ const WebhookSection: React.FC<WebhoookSectionProps> = ({ pac }) => {
   const [controllerUrl, setControllerUrl] = React.useState('');
   const [webhookSecret, setWebhookSecret] = React.useState('');
   const { t } = useTranslation();
+  const isStrategy = values?.pac?.formType === 'strategy';
+  const widthAdjustment = isStrategy
+    ? values?.pac?.repository?.githubAppAvailable
+    : values?.githubAppAvailable;
```

Comment:
@Lucifergene instead of checking which form it is every time and based on that use/update the values, you can use a ` formContextField` prop similar to how its being used in ` ImageStreamDropdown` and other components so that this component can be used in any form later.